plugin-build/src/test/kotlin/io/sentry/android/gradle/integration/BaseSentryPluginTest.kt

```diff
@@ -108,12 +108,6 @@ abstract class BaseSentryPluginTest(
                 }
               }
             }
-
-                // unlock transforms because we're running tests in parallel therefore they may conflict
```

Comment:
This was causing configuration cache to be invalidated sometimes, like [here](https://github.com/getsentry/sentry-android-gradle-plugin/actions/runs/15877905474/job/44770117956), so our tests that test the config cache reuse were failing. So we now just run this outside the tests gradle build